lib/pmdtester/cmd.rb

```diff
@@ -5,20 +5,36 @@
 module PmdTester
   # Containing the common method for executing shell command
   class Cmd
+    extend PmdTester
     def self.execute(cmd)
-      puts cmd
+      logger.info "execute command '#{cmd}'"
```

Comment:
This should be debug. We probably have in the calling code already a log.info, explaining that we will execute some command (not the exact command line, but that's part of the debug logging then).